static/js/controllers/messages-content.js

```diff
@@ -172,8 +172,19 @@ var _ = require('underscore'),
       };
 
       $scope.addRecipients = function(to) {
+        var realTo;
+        if (to.facility) {
+          realTo = to.facility;
+        } else if (to.contact) {
+          if (to.contact._id) {
+            realTo = to.contact;
+          } else if (to.contact.name) {
+            realTo = to.contact.name;
```

Comment:
This is where types would be really helpful, to help implicitly document structures. This is the structure of a raw phone number with no contact attached:
![image](https://cloud.githubusercontent.com/assets/583851/18046710/074b24f8-6dd1-11e6-9884-47849953d5f2.png)
I'll add a comment indicating this is the case the branch is dealing with.